src/smt/command.cpp

```diff
@@ -385,7 +385,7 @@ std::string PopCommand::getCommandName() const { return "pop"; }
 /* class CheckSatCommand                                                      */
 /* -------------------------------------------------------------------------- */
 
-CheckSatCommand::CheckSatCommand() : d_expr() {}
+CheckSatCommand::CheckSatCommand() : d_expr(), d_inUnsatCore() {}
```

Comment:
Seems like d_inUnsatCore is not used in CheckSatCommand (and in CheckSatAssumingCommand). I can submit a PR to remove it.

Reply:
I went ahead and removed these.